datasets.py

```diff
@@ -64,6 +64,22 @@ def build_dataset(is_train, args):
         root = os.path.join(args.data_path, 'train' if is_train else 'val')
         dataset = datasets.ImageFolder(root, transform=transform)
         nb_classes = 1000
+        
+        if is_proxy: # create dataset with 1 image per classes
```

Comment:
The function here is to create a proxy dataset that contain 1000 images (1 image per classes). 
When running the greedy supernet training, we will use this proxy dataset as a benchmark